executor/index_lookup_join.go

```diff
@@ -151,7 +153,10 @@ func (e *IndexLookUpJoin) Close() error {
 // Step6: fetch a batch of sorted "inner rows" based on the request rows.
 // Step7: do merge join on the **sorted** outer and inner rows.
 func (e *IndexLookUpJoin) Next() (Row, error) {
-	for ; e.resultCursor == len(e.resultBuffer); e.resultCursor = 0 {
+	for ; e.resultCursor == len(e.resultBuffer); e.resultCursor, e.curBatchSize = 0, e.curBatchSize*2 {
+		if e.curBatchSize > e.batchSize {
```

Comment:
`curBatchSize` may overflow of max int

Reply:
Should we set a max value for curBatchSize?

Reply:
I think we should set a max value for e.batchSize to make it less than max_int_64/2

Reply:
the max value of `e.batchSize` can be `e.batchSize`, `max_int_64/2` is too large